OrangeDesignSystemDemo/OrangeDesignSystemDemo.xcodeproj/project.pbxproj

```diff
@@ -994,6 +1023,24 @@
 			shellPath = /bin/sh;
 			shellScript = "# Periphery action to look for dead code\nperiphery scan --strict --workspace OrangeDesignSystemDemo.xcworkspace --schemes OrangeDesignSystemDemo --targets OrangeDesignSystemDemo --format xcode\n";
 		};
+		51EE08912B0F7464003A254A /* Check JSON files format */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+			);
+			name = "Check JSON files format";
+			outputFileListPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "# Check if AppNews.json file is a conform JSON file and throw error if not to prevent to build and ship with corrupted file\nAppNewsFileFolder=\"${PROJECT_DIR}/OrangeDesignSystemDemo/Resources/\"\n\nAppNewsFileBase=\"$AppNewsFileFolder/Base.lproj/AppNews.json\"\nif cat \"$AppNewsFileBase\" | jq >/dev/null 2>&1; then\n    echo \"AppNews.json (Base) seems good\"\nelse\n    echo \"error: AppNews.json (Base) is not conform to JSON format\"\n    exit 1\nfi\n\nAppNewsFileBase=\"$AppNewsFileFolder/en.lproj/AppNews.json\"\nif cat \"$AppNewsFileBase\" | jq >/dev/null 2>&1; then\n    echo \"AppNews.json (en) seems good\"\nelse\n    echo \"error: AppNews.json (en) is not conform to JSON format\"\n    exit 1    \nfi\n\nAppNewsFileBase=\"$AppNewsFileFolder/fr.lproj/AppNews.json\"\nif cat \"$AppNewsFileBase\" | jq >/dev/null 2>&1; then\n    echo \"AppNews.json (fr) seems good\"\nelse\n    echo \"error: AppNews.json (fr) is not conform to JSON format\"\n    exit 1    \nfi\n";
```

Comment:
Add entry in DEVELOPER.md to enumerated all build phases scripts because we sil forget them.
An becare full is will increase the build time...

Reply:
The operations are quite smooth and efficient, so I am not sure it will increase buid time so much.
I agree about the need to update the DEVELOPER.md file.